{"name": "test_loginfail", "status": "broken", "statusDetails": {"message": "AttributeError: 'CustomLogger' object has no attribute 'info'", "trace": "self = <pages.LoginPage.LoginPage object at 0x0000016AA0B93750>, text = '9999999999', locatorValue = '//android.widget.EditText'\nlocatorType = 'xpath'\n\n    def SendKeys(self,text, locatorValue, locatorType=\"id\"):\n        element = None\n        try:\n            locatorType = locatorType.lower()\n            element = self.waitForElement(locatorValue, locatorType)\n            element.send_keys(text)\n>           self.log.info(\"Send a text with locator type\" + locatorType + \"with locator value\" + locatorValue)\nE           AttributeError: 'CustomLogger' object has no attribute 'info'\n\n..\\base\\BasePage.py:99: AttributeError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.LoginTest.LoginTest testMethod=test_loginfail>\n\n    @pytest.mark.run(order=2)\n    def test_loginfail(self):\n>       self.lp.enterPhoneNumber()\n\nLoginTest.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\LoginPage.py:23: in enterPhoneNumber\n    self.SendKeys(\"9999999999\",self._enterPhoneNumer,\"xpath\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.LoginPage.LoginPage object at 0x0000016AA0B93750>, text = '9999999999', locatorValue = '//android.widget.EditText'\nlocatorType = 'xpath'\n\n    def SendKeys(self,text, locatorValue, locatorType=\"id\"):\n        element = None\n        try:\n            locatorType = locatorType.lower()\n            element = self.waitForElement(locatorValue, locatorType)\n            element.send_keys(text)\n            self.log.info(\"Send a text with locator type\" + locatorType + \"with locator value\" + locatorValue)\n    \n        except:\n>           self.log.info(\"Unable to send the text with locator type\" + locatorType + \"with locator value\" + locatorValue)\nE           AttributeError: 'CustomLogger' object has no attribute 'info'\n\n..\\base\\BasePage.py:102: AttributeError"}, "start": 1739959788078, "stop": 1739959789201, "uuid": "2b8694fd-0071-4737-8106-f0fc16b03646", "historyId": "2e9d26c442f09a305fb336d897266e93", "testCaseId": "2e9d26c442f09a305fb336d897266e93", "fullName": "LoginTest.LoginTest#test_loginfail", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "suite", "value": "LoginTest"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "Chinmayi"}, {"name": "thread", "value": "8476-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "LoginTest"}]}